webapp/src/test/resources/rmap.properties

```diff
@@ -0,0 +1,24 @@
+#overridden test properties
+rmapweb.google-oauth-enabled=true
```

Comment:
Maybe update `integration/src/main/resources/rmap.properties` and remove any old auth properties and add sample (but commented out as appropriate) updated auth properties?

Reply:
Added the properties but commented out.  They only need to be overridden if enabled.